jbpm-designer-backend/src/test/java/org/jbpm/designer/repository/VFSRepositoryDefaultFileSystemTest.java

```diff
@@ -460,6 +461,21 @@ public void testAssetExists() throws NoSuchFileException {
         assertTrue(assetExists);
     }
 
+    @Test(expected = FileAlreadyExistsException.class)
+    public void testCreateAssetAlreadyExists() {
+        Repository repository = new VFSRepository(producer.getIoService());
+        ((VFSRepository) repository).setDescriptor(descriptor);
+
+        AssetBuilder builder = AssetBuilderFactory.getAssetBuilder(Asset.AssetType.Text);
+        builder.content("simple content")
+                .type("txt")
+                .name("test")
+                .location("/");
+
+        repository.createAsset(builder.getAsset());
+        repository.createAsset(builder.getAsset());
```

Comment:
Please use assertj's [assertThatThrownBy](https://github.com/joel-costigliola/assertj-examples/blob/master/assertions-examples/src/test/java/org/assertj/examples/ExceptionAssertionsExamples.java#L202-L208) to make the intention clearer (and check the exception error message too). The same in the whole PR ensemble.